src/app/home/modules/about/containers/about/about.component.html

```diff
@@ -42,18 +44,27 @@ <h4>Workshop day</h4>
 			<strong>March 23 is a multi-track workshop day</strong>, where attendees can learn about the latest Angular
 			developments.
 		</p>
+		<p>
+			More details? <a href="https://ng-be.org/workshop-day" target="_blank">Click here for our full workshop planning</a>
+		</p>
 
 		<h4>Conference day</h4>
 		<p>
 			<strong>March 24 is a single-track conference day</strong>. Attendees will be able to listen to talks from
 			world-renowned Angular experts while enjoying delicious foods and drinks.
 		</p>
-
-		<h4>App credits</h4>
 		<p>
-			&bull; <a href="https://www.twitter.com/SamVerschueren" target="_blank">Sam Verschueren</a><br />
-			&bull; <a href="https://www.twitter.com/fulvio_the1st" target="_blank">Fulvio Gentile</a><br />
-			&bull; <a href="https://www.twitter.com/c4rniatto" target="_blank">Mateus Carniatto</a>
+			More details? <a href="https://ng-be.org/conference-day" target="_blank">Click here for our full conference planning</a>
```

Comment:
Should this just link to the schedule page in the app instead of going to an external link?

```suggestion
			More details? <a href="https://ng-be.org/conference-day" target="_blank">Click here for our full conference planning</a>.
```

Reply:
Ok! That sounds good!